plugins/frontend/backstage-plugin-prometheus/src/types.ts

```diff
@@ -79,3 +79,7 @@ export type PrometheusRuleResponse = {
   };
   status: 'success' | 'error';
 };
+
+export type CallbackProp = {
+  callback?: (arg: PrometheusDisplayableAlert | undefined) => void;
+};
```

Comment:
1. What do you think of when you read `CallbackProp`? Do you instantly recognize this is something that happens on click on a row and only then?
2. Passing a prop which is an object that has one and only property doesn't look like the most efficient way. Why is there a `CallbackProp` when only the `CallbackProp.callback` matters? There's no other use for this, so the nesting does not bring any value

```suggestion
export type onRowClick = (arg: PrometheusDisplayableAlert | undefined) => void;
```

Reply:
@tumido Thanks for suggestion :+1: I have modified code to have a type of `OnRowClick` wdyt?